common/thanos/templates/alerts/_thanos-compactor.alerts.tpl

```diff
@@ -3,83 +3,109 @@
 groups:
 - name: thanos-compactor.alerts
   rules:
-    - alert: ThanosCompactHalted
-      expr: thanos_compact_halted{app="thanos-compactor", prometheus="{{ include "thanos.name" . }}"} == 1
+    - alert: ThanosCompactMultipleRunning
+      expr: sum by (thanos) (up{job=~".*thanos.*compact.*", thanos="{{ include "thanos.name" . }}"}) > 1
```

Comment:
Why don't we sum by `job` here?

Reply:
sum by `job` would mean that `$labels.thanos` is not resolved in the meta.